WordPress/WordPressTest/Me/App Settings/Boolean User Defaults/BooleanUserDefaultsDebugViewModelTests.swift

```diff
@@ -0,0 +1,127 @@
+import XCTest
+@testable import WordPress
+
+class BooleanUserDefaultsDebugViewModelTests: CoreDataTestCase {
+
+    var viewModel: BooleanUserDefaultsDebugViewModel!
+    var mockPersistentRepository: InMemoryUserDefaults!
+
+    override func setUp() {
+        super.setUp()
+        mockPersistentRepository = InMemoryUserDefaults()
+        viewModel = BooleanUserDefaultsDebugViewModel(coreDataStack: contextManager,
+                                                      persistentRepository: mockPersistentRepository)
+    }
+
+    override func tearDown() {
+        viewModel = nil
+        mockPersistentRepository = nil
+        super.tearDown()
+    }
+
+    func testLoadUserDefaults_WithOtherSection() {
+            // Given
+            mockPersistentRepository.set(true, forKey: "entry1")
+
+            // When
+            viewModel.load()
+
+            // Then
+        XCTAssertTrue(viewModel.userDefaultsSections["Other"]?["entry1"]?.value ?? false)
```

Comment:
This can be a bit more readable:
```suggestion
        let value = try XCTUnwrap(viewModel.userDefaultsSections["Other"]?["entry1"]?.value)
        XCTAssertTrue(value)
```